tests/roles/ovn_adoption/defaults/main.yaml

```diff
@@ -34,7 +34,7 @@ ovn_nic_mapping_patch: |
       template:
         ovnController:
           nicMappings:
-            baremetal: baremetal
+            ironic: baremetal
```

Comment:
Is this change needs to be fixed somewhere in documentation too?

Reply:
It is a variable, we have no way of knowing what choices the users would have done when naming their physical networks and interfaces. For the human operator doing adoption "ironic" here must match the name used in the source OSP17 cloud, and "baremetal" (the interface) must match the name of the interface defined in the NNCP for the OCP cluster.

I did a quick grep, and don't see anywhere in docs that need a change.

Reply:
thank you for clarifying